ui/apps/platform/src/Containers/Collections/CollectionsTable.tsx

```diff
@@ -40,8 +70,39 @@ function CollectionsTable({ collections, getSortParams, hasWriteAccess }: Collec
         });
     }
 
+    const onSearchInputChange = useMemo(
+        () => debounce((value: string) => setSearchFilter({ Collection: value }), TYPING_DELAY),
+        [setSearchFilter]
+    );
+
+    const searchValue = Array.isArray(searchFilter.Collection)
+        ? searchFilter.Collection.join('+')
+        : searchFilter.Collection;
+
     return (
         <>
+            <Toolbar>
+                <ToolbarContent>
+                    <ToolbarItem variant="search-filter" className="pf-u-flex-grow-1">
+                        <SearchInput
+                            aria-label="Search by name"
+                            placeholder="Search by name..."
```

Comment:
3 periods in placeholder seems to be a new trend. Only other occurrence is in policy categories.
* Placeholder examples for `SearchInput` element do not include it:
    https://www.patternfly.org/v4/components/search-input
* UX writing guidelines do not include it:
    https://www.patternfly.org/v4/ux-writing/punctuation#ellipses-...